### DISCUSSION TITLE
How do I stop a dropdown menu from closing after clicking an option

### DISCUSSION THREAD

**Discussion by StephenSHorton** [radix-ui/primitives#2391] [Help] [answered]
Can't for the life of me figure this out. I could just put my own state in for the open and closing but how do I let the default behavior of clicking outside take over after that?

**Comment by StephenSHorton** [comment]
Just going to use the onPointerDownOutside

**Comment by benoitgrelard** [comment, marked as answer]
Hi @StephenSHorton, I assume you mean when selecting an item? If so, the correct answer is to call `event.preventDefault` in `onSelect`.

![CleanShot 2024-02-06 at 14 52 16@2x](https://github.com/radix-ui/primitives/assets/1539897/27f2509a-76fe-4185-a37f-8ccab1aa35ff)

**Reply by StephenSHorton** [reply]
@benoitgrelard I appreciate so much that you took time to answer! 5 months later I actually got ahold of that answer luckily. Since you're here, thought I'd say that I LOVE Radix UI. It's been a pleasure working with it in my project.
